Makefile

```diff
@@ -10,6 +10,9 @@ default: check
 
 check:
 	node builders/verification-compelling-verify.js _includes/verification-compelling-intro.html
+
+check-dafny:
```

Comment:
Could you please explain this goal in the README.md?

Reply:
I have made a minor fix to `verification-compelling-verify.js`, so it can now also run as a child process as part of the CI and there is no reason to have a separate `check-dafny` goal anymore, so no reason to update README either